lib/src/screens/memory/memory_service.dart

```diff
@@ -186,31 +190,8 @@ class MemoryService {
         ..selectedAppEventIndex = eventIndex
         ..modelDataSearchTextIndices = ListBuilder([])
         ..modelDataSearchText = ''
-        ..scroll = false,
+        ..scroll = true,
```

Comment:
why true ?

Reply:
ah I was testing why it was false and forgot to revert 😛